Tests fail on Python 2.7

As of now, we don't want to drop support for Python 2.7 before Flask does. When we fixed the CI recently, we see the build for 2.7 is broken though - https://github.com/Frozen-Flask/Frozen-Flask/actions. Any help with getting the support for 2.7 fixed is very appreciated!

```
.F.E....FF.FE........F.E....FF.FE........F.E....FF.FE........F.E.....FF.FE........F.E....FF.FE.........F.E....FF.FE........F.E....FF.FE........F.E....FF.FE................................
======================================================================
ERROR: test_contents (__main__.TestBaseURL)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 226, in test_contents
    content = read_file(filename)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 53, in read_file
    with open(filename, 'rb') as fd:
IOError: [Errno 2] No such file or directory: u'/tmp/tmpPEPju5/page/I l\xf8v\xeb Unicode/index.html'

======================================================================
ERROR: test_transitivity (__main__.TestBaseURL)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 273, in test_transitivity
    freezer2.freeze()
  File "flask_frozen/__init__.py", line 205, in freeze
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 205, in <genexpr>
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 189, in freeze_yield
    new_filename = self._build_one(url, last_modified)
  File "flask_frozen/__init__.py", line 338, in _build_one
    % (response.status, url))
ValueError: Unexpected status '404 NOT FOUND' on URL /page/I l\xf8v\xeb Unicode/

======================================================================
ERROR: test_contents (__main__.TestFreezer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 226, in test_contents
    content = read_file(filename)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 53, in read_file
    with open(filename, 'rb') as fd:
IOError: [Errno 2] No such file or directory: u'/tmp/tmpq7Ur6V/page/I l\xf8v\xeb Unicode/index.html'

======================================================================
ERROR: test_transitivity (__main__.TestFreezer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 273, in test_transitivity
    freezer2.freeze()
  File "flask_frozen/__init__.py", line 205, in freeze
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 205, in <genexpr>
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 189, in freeze_yield
    new_filename = self._build_one(url, last_modified)
  File "flask_frozen/__init__.py", line 338, in _build_one
    % (response.status, url))
ValueError: Unexpected status '404 NOT FOUND' on URL /page/I l\xf8v\xeb Unicode/

======================================================================
ERROR: test_contents (__main__.TestInitApp)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 226, in test_contents
    content = read_file(filename)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 53, in read_file
    with open(filename, 'rb') as fd:
IOError: [Errno 2] No such file or directory: u'/tmp/tmpNrhtbO/page/I l\xf8v\xeb Unicode/index.html'

======================================================================
ERROR: test_transitivity (__main__.TestInitApp)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 273, in test_transitivity
    freezer2.freeze()
  File "flask_frozen/__init__.py", line 205, in freeze
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 205, in <genexpr>
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 189, in freeze_yield
    new_filename = self._build_one(url, last_modified)
  File "flask_frozen/__init__.py", line 338, in _build_one
    % (response.status, url))
ValueError: Unexpected status '404 NOT FOUND' on URL /page/I l\xf8v\xeb Unicode/

======================================================================
ERROR: test_contents (__main__.TestLastModifiedGenerator)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 226, in test_contents
    content = read_file(filename)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 53, in read_file
    with open(filename, 'rb') as fd:
IOError: [Errno 2] No such file or directory: u'/tmp/tmpvy6yKQ/page/I l\xf8v\xeb Unicode/index.html'

======================================================================
ERROR: test_transitivity (__main__.TestLastModifiedGenerator)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 273, in test_transitivity
    freezer2.freeze()
  File "flask_frozen/__init__.py", line 205, in freeze
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 205, in <genexpr>
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 189, in freeze_yield
    new_filename = self._build_one(url, last_modified)
  File "flask_frozen/__init__.py", line 338, in _build_one
    % (response.status, url))
ValueError: Unexpected status '404 NOT FOUND' on URL /page/I l\xf8v\xeb Unicode/

======================================================================
ERROR: test_contents (__main__.TestNonexsistentDestination)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 226, in test_contents
    content = read_file(filename)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 53, in read_file
    with open(filename, 'rb') as fd:
IOError: [Errno 2] No such file or directory: u'/tmp/tmp4llyTv/frozen/htdocs/page/I l\xf8v\xeb Unicode/index.html'

======================================================================
ERROR: test_transitivity (__main__.TestNonexsistentDestination)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 273, in test_transitivity
    freezer2.freeze()
  File "flask_frozen/__init__.py", line 205, in freeze
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 205, in <genexpr>
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 189, in freeze_yield
    new_filename = self._build_one(url, last_modified)
  File "flask_frozen/__init__.py", line 338, in _build_one
    % (response.status, url))
ValueError: Unexpected status '404 NOT FOUND' on URL /page/I l\xf8v\xeb Unicode/

======================================================================
ERROR: test_contents (__main__.TestRelativeUrlFor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 226, in test_contents
    content = read_file(filename)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 53, in read_file
    with open(filename, 'rb') as fd:
IOError: [Errno 2] No such file or directory: u'/tmp/tmpM6Z_ha/page/I l\xf8v\xeb Unicode/index.html'

======================================================================
ERROR: test_transitivity (__main__.TestRelativeUrlFor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 273, in test_transitivity
    freezer2.freeze()
  File "flask_frozen/__init__.py", line 205, in freeze
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 205, in <genexpr>
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 189, in freeze_yield
    new_filename = self._build_one(url, last_modified)
  File "flask_frozen/__init__.py", line 338, in _build_one
    % (response.status, url))
ValueError: Unexpected status '404 NOT FOUND' on URL /page/I l\xf8v\xeb Unicode/

======================================================================
ERROR: test_contents (__main__.TestServerName)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 226, in test_contents
    content = read_file(filename)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 53, in read_file
    with open(filename, 'rb') as fd:
IOError: [Errno 2] No such file or directory: u'/tmp/tmpBMwwTR/page/I l\xf8v\xeb Unicode/index.html'

======================================================================
ERROR: test_transitivity (__main__.TestServerName)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 273, in test_transitivity
    freezer2.freeze()
  File "flask_frozen/__init__.py", line 205, in freeze
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 205, in <genexpr>
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 189, in freeze_yield
    new_filename = self._build_one(url, last_modified)
  File "flask_frozen/__init__.py", line 338, in _build_one
    % (response.status, url))
ValueError: Unexpected status '404 NOT FOUND' on URL /page/I l\xf8v\xeb Unicode/

======================================================================
ERROR: test_contents (__main__.TestStaticIgnore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 226, in test_contents
    content = read_file(filename)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 53, in read_file
    with open(filename, 'rb') as fd:
IOError: [Errno 2] No such file or directory: u'/tmp/tmp4jcuk9/page/I l\xf8v\xeb Unicode/index.html'

======================================================================
ERROR: test_transitivity (__main__.TestStaticIgnore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 273, in test_transitivity
    freezer2.freeze()
  File "flask_frozen/__init__.py", line 205, in freeze
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 205, in <genexpr>
    return set(page.url for page in self.freeze_yield())
  File "flask_frozen/__init__.py", line 189, in freeze_yield
    new_filename = self._build_one(url, last_modified)
  File "flask_frozen/__init__.py", line 338, in _build_one
    % (response.status, url))
ValueError: Unexpected status '404 NOT FOUND' on URL /page/I l\xf8v\xeb Unicode/

======================================================================
FAIL: test_built_urls (__main__.TestBaseURL)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 217, in test_built_urls
    self.assertEqual(set(urls), set(self.expected_output))
AssertionError: Items in the first set but not the second:
u'/page/I l\xc3\xb8v\xc3\xab Unicode/'
Items in the second set but not the first:
u'/page/I l\xf8v\xeb Unicode/'

======================================================================
FAIL: test_ignore_pattern (__main__.TestBaseURL)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 236, in test_ignore_pattern
    self._extra_files(ignore=['extraa'], removed=True)   # Not a match
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_nothing_else_matters (__main__.TestBaseURL)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 230, in test_nothing_else_matters
    self._extra_files(removed=True)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_something_else_matters (__main__.TestBaseURL)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 233, in test_something_else_matters
    self._extra_files(remove_extra=False, removed=False)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_built_urls (__main__.TestFreezer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 217, in test_built_urls
    self.assertEqual(set(urls), set(self.expected_output))
AssertionError: Items in the first set but not the second:
u'/page/I l\xc3\xb8v\xc3\xab Unicode/'
Items in the second set but not the first:
u'/page/I l\xf8v\xeb Unicode/'

======================================================================
FAIL: test_ignore_pattern (__main__.TestFreezer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 236, in test_ignore_pattern
    self._extra_files(ignore=['extraa'], removed=True)   # Not a match
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_nothing_else_matters (__main__.TestFreezer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 230, in test_nothing_else_matters
    self._extra_files(removed=True)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_something_else_matters (__main__.TestFreezer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 233, in test_something_else_matters
    self._extra_files(remove_extra=False, removed=False)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_built_urls (__main__.TestInitApp)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 217, in test_built_urls
    self.assertEqual(set(urls), set(self.expected_output))
AssertionError: Items in the first set but not the second:
u'/page/I l\xc3\xb8v\xc3\xab Unicode/'
Items in the second set but not the first:
u'/page/I l\xf8v\xeb Unicode/'

======================================================================
FAIL: test_ignore_pattern (__main__.TestInitApp)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 236, in test_ignore_pattern
    self._extra_files(ignore=['extraa'], removed=True)   # Not a match
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_nothing_else_matters (__main__.TestInitApp)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 230, in test_nothing_else_matters
    self._extra_files(removed=True)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_something_else_matters (__main__.TestInitApp)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 233, in test_something_else_matters
    self._extra_files(remove_extra=False, removed=False)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_built_urls (__main__.TestLastModifiedGenerator)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 217, in test_built_urls
    self.assertEqual(set(urls), set(self.expected_output))
AssertionError: Items in the first set but not the second:
u'/page/I l\xc3\xb8v\xc3\xab Unicode/'
Items in the second set but not the first:
u'/page/I l\xf8v\xeb Unicode/'

======================================================================
FAIL: test_ignore_pattern (__main__.TestLastModifiedGenerator)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 236, in test_ignore_pattern
    self._extra_files(ignore=['extraa'], removed=True)   # Not a match
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_nothing_else_matters (__main__.TestLastModifiedGenerator)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 230, in test_nothing_else_matters
    self._extra_files(removed=True)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_something_else_matters (__main__.TestLastModifiedGenerator)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 233, in test_something_else_matters
    self._extra_files(remove_extra=False, removed=False)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_built_urls (__main__.TestNonexsistentDestination)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 217, in test_built_urls
    self.assertEqual(set(urls), set(self.expected_output))
AssertionError: Items in the first set but not the second:
u'/page/I l\xc3\xb8v\xc3\xab Unicode/'
Items in the second set but not the first:
u'/page/I l\xf8v\xeb Unicode/'

======================================================================
FAIL: test_ignore_pattern (__main__.TestNonexsistentDestination)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 236, in test_ignore_pattern
    self._extra_files(ignore=['extraa'], removed=True)   # Not a match
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_nothing_else_matters (__main__.TestNonexsistentDestination)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 230, in test_nothing_else_matters
    self._extra_files(removed=True)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_something_else_matters (__main__.TestNonexsistentDestination)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 233, in test_something_else_matters
    self._extra_files(remove_extra=False, removed=False)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_built_urls (__main__.TestRelativeUrlFor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 217, in test_built_urls
    self.assertEqual(set(urls), set(self.expected_output))
AssertionError: Items in the first set but not the second:
u'/page/I l\xc3\xb8v\xc3\xab Unicode/'
Items in the second set but not the first:
u'/page/I l\xf8v\xeb Unicode/'

======================================================================
FAIL: test_ignore_pattern (__main__.TestRelativeUrlFor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 236, in test_ignore_pattern
    self._extra_files(ignore=['extraa'], removed=True)   # Not a match
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_nothing_else_matters (__main__.TestRelativeUrlFor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 230, in test_nothing_else_matters
    self._extra_files(removed=True)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_something_else_matters (__main__.TestRelativeUrlFor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 233, in test_something_else_matters
    self._extra_files(remove_extra=False, removed=False)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_built_urls (__main__.TestServerName)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 217, in test_built_urls
    self.assertEqual(set(urls), set(self.expected_output))
AssertionError: Items in the first set but not the second:
u'/page/I l\xc3\xb8v\xc3\xab Unicode/'
Items in the second set but not the first:
u'/page/I l\xf8v\xeb Unicode/'

======================================================================
FAIL: test_ignore_pattern (__main__.TestServerName)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 236, in test_ignore_pattern
    self._extra_files(ignore=['extraa'], removed=True)   # Not a match
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_nothing_else_matters (__main__.TestServerName)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 230, in test_nothing_else_matters
    self._extra_files(removed=True)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_something_else_matters (__main__.TestServerName)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 233, in test_something_else_matters
    self._extra_files(remove_extra=False, removed=False)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/main.js',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_built_urls (__main__.TestStaticIgnore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 217, in test_built_urls
    self.assertEqual(set(urls), set(self.expected_output))
AssertionError: Items in the first set but not the second:
u'/page/I l\xc3\xb8v\xc3\xab Unicode/'
Items in the second set but not the first:
u'/page/I l\xf8v\xeb Unicode/'

======================================================================
FAIL: test_ignore_pattern (__main__.TestStaticIgnore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 236, in test_ignore_pattern
    self._extra_files(ignore=['extraa'], removed=True)   # Not a match
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_nothing_else_matters (__main__.TestStaticIgnore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 230, in test_nothing_else_matters
    self._extra_files(removed=True)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/style.css',
   u'where_am_i/index.html']

======================================================================
FAIL: test_something_else_matters (__main__.TestStaticIgnore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 233, in test_something_else_matters
    self._extra_files(remove_extra=False, removed=False)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 247, in _extra_files
    self.assertFilenamesEqual(walk_directory(dest), expected_files)
  File "/home/runner/work/Frozen-Flask/Frozen-Flask/flask_frozen/tests.py", line 199, in assertFilenamesEqual
    self.assertEqual(set1, set2)
AssertionError: Lists differ: [u'admin/css/style.css', u'adm... != [u'admin/css/style.css', u'adm...

First differing element 4:
u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html'
u'page/I l\xf8v\xeb Unicode/index.html'

  [u'admin/css/style.css',
   u'admin/index.html',
   u'favicon.ico',
   u'index.html',
-  u'page/I l\xc3\xb8v\xc3\xab Unicode/index.html',
?              ^^^^^    ^^^^^

+  u'page/I l\xf8v\xeb Unicode/index.html',
?              ^    ^

   u'page/foo/index.html',
   u'page/octothorp/index.html',
   u'product_0/index.html',
   u'product_1/index.html',
   u'product_2/index.html',
   u'product_3/index.html',
   u'product_4/index.html',
   u'product_5/index.html',
   u'redirect/index.html',
   u'robots.txt',
   u'static/favicon.ico',
   u'static/style.css',
   u'where_am_i/index.html']

----------------------------------------------------------------------
Ran 187 tests in 8.531s

FAILED (failures=32, errors=16)
```

--- a/flask_frozen/__init__.py
+++ b/flask_frozen/__init__.py
@@ -35,8 +35,8 @@ except ImportError:
     from collections import Mapping  # Python 2.7
 
 try:
-    from urllib import unquote
     from urlparse import urlsplit
+    from werkzeug.urls import url_unquote as unquote
 except ImportError:  # Python 3
     from urllib.parse import urlsplit, unquote
 

